apps/dapp-console-api/migrations/0003_gorgeous_doctor_doom.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE IF NOT EXISTS "contracts" (
+	"created_at" timestamp with time zone DEFAULT now() NOT NULL,
+	"updated_at" timestamp with time zone DEFAULT now() NOT NULL,
+	"id" uuid PRIMARY KEY DEFAULT gen_random_uuid() NOT NULL,
+	"entity_id" uuid,
+	"chain_id" integer,
+	"app_id" uuid,
+	"name" varchar NOT NULL,
+	"contract_address" varchar,
+	"deployer_address" varchar,
+	"state" varchar DEFAULT 'not_verified' NOT NULL
+);
+--> statement-breakpoint
+DROP INDEX IF EXISTS "address_idx";--> statement-breakpoint
```

Comment:
you might need to regen this so it doesn't worry about this idx

Reply:
weird, not sure how that happened. this should be fixed now.